eve/main.yml

```
@@ -1103,7 +1109,9 @@ stages:
             ISO_PATH: /tmp/metalk8s.iso
       - ShellCommand:
           <<: *bootstrap_config
-          env: *_env_version_prev
+          env:
+            <<: *_env_version_prev
+            DEBUG: "%(prop:metalk8s_debug)s"
```

No default value to fallback on, as you did above:
```suggestion
            DEBUG: "%(prop:metalk8s_debug:-false)s"
```

---

Done